spack.yaml

```
@@ -0,0 +1,83 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  specs:
+    - access-esm1p5@git.2024.05.0
+  packages:
+    # Direct ACCESS-NRI dependencies
+    # Note: mom5, oasis3-mct have branch-specific logic and hence can't use
+    # the usual '@git.DATE' format.
```

```suggestion
    # the usual '@git.DATE' ([calver](https://calver.org)) format.
```

Doesn't the `um` also use the branch specification?

---

Done in https://github.com/ACCESS-NRI/ACCESS-ESM1.5/pull/5/commits/34ed057cfc8f8dad6819b4194d1ec4caa5ce52e4